cpp/oneapi/dal/algo/basic_statistics/test/fixture.hpp

```diff
@@ -100,6 +105,17 @@ class basic_statistics_test : public te::crtp_algo_fixture<TestType, Derived> {
         check_for_exception_for_non_requested_results(compute_mode, compute_result);
     }
 
+    void csr_general_checks(const te::csr_table_builder& data, bs::result_option_id compute_mode) {
+        const auto desc =
+            bs::descriptor<float_t, basic_statistics::method::sparse>{}.set_result_options(
+                compute_mode);
+        const auto csr_table = data.build_csr_table(this->get_policy());
+        const auto dense_table = data.build_dense_table();
```

Comment:
For now it is Ok to test it like this.
But further it would be good to be able to test bigger CSR tables that cannot be converted to dense tables due to memory limitations.